workload/src/Samsung.Tizen.Sdk/Sdk/AutoImport.props

```diff
@@ -18,6 +18,7 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
+    <TizenWorkloadInstalled Condition="'$(TizenWorkloadInstalled)' == ''">true</TizenWorkloadInstalled>
```

Comment:
Did you check if this property works well outside? for example, Directory.Build.props in maui project

Reply:
@mattleibow I'd like to update the condition of `<IncludeTizenTargetFrameworks>` in maui, do you have any additional comments?

- Before
```xml
<IncludeTizenTargetFrameworks Condition="'$(CI)' == 'true' or '$(TF_BUILD)' == 'true' or
             Exists('$(DOTNET_ROOT)\sdk-manifests\$(DotNetVersionBand)\samsung.net.sdk.tizen\WorkloadManifest.json') or
             Exists('$(ProgramFiles)\dotnet\sdk-manifests\$(DotNetVersionBand)\samsung.net.sdk.tizen\WorkloadManifest.json')">true</IncludeTizenTargetFrameworks>
  </PropertyGroup>
```

- After
```xml
<IncludeTizenTargetFrameworks Condition="'$(TizenWorkloadInstalled)' == 'true' > true </PropertyGroup>
```

Plus, I can use this on template projects of maui project as below.
```xml
<TargetFrameworks Condition="'$(TizenWorkloadInstalled)' == 'true' >$(TargetFrameworks);DOTNET_TFM-tizen</TargetFrameworks>
```

Reply:
Yes, I added this property by editing `C:\Program Files\dotnet\packs\Samsung.Tizen.Sdk\7.0.100-preview.13.30\Sdk\AutoImport.props` directly. instead of re-installing the whole Tizen workload.

I tried using the property at `Directory.Build.props` and I could see this value is available. I will also try it in some more places like template's project.

Reply:
To me, in a same way of testing method, `$(TizenWorkloadInstalled)` looks available on `Microsoft.Maui.Templates` project.

Reply:
it should be available not only for apps but also for building maui.
When I tested by using CLI, the apps were not a problem, but it was a problem when building core, controls, etc.

Reply:
Yes, I meant using it for core, controls, etc were ok too. Can you share how you test it?

Reply:
@JoonghyunCho Why this property has a condition? `Condition="'$(TizenWorkloadInstalled)' == ''"` Is there any case that it is set to `false` outside?

Reply:
No, I just added it just in case later I face some cases I need to set to `false`. Maybe that is not a normal situation. I can remove it.